include/seastar/rpc/rpc.hh

```diff
@@ -623,6 +623,14 @@ public:
             f(*c.second);
         }
     }
+    /**
+     * Abort the given connection, causing it to stop receiving any further messages.
+     * It's safe to abort a connection from an RPC handler running on that connection.
+     * Does nothing if there is no connection with the given ID on this server.
```

Comment:
This is a dangerous statement to make, because it places the burden of lifetime tracking on the server. IIRC now it's handled by never recycling connection_id:s. It could also be handled by making connection_id a weak pointer to a connection.

I guess it's okay.